### ISSUE TITLE
Enhancement: Add a class named Conveter that will handle the ConversionType property and also the other object converters.

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDB#417]
As part of the code organization for the mappers (ie: TypeMapper), we need to create a class that will handle the ConversionType property based on its own purpose.

This class will handle that value.

So instead of calling the following.

```csharp
TypeMapper.ConversionType = ConversionType.Automatic;
```

You must call this instead.

```csharp
Converter.ConversionType = ConversionType.Automatic;
```

**Comment by mikependon** [comment]
Codes has been checked-in.

**Comment by mikependon** [comment]
Will be available at RepoDb version >= v1.10.0-alpha1.
